Form/RequestPassword.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Form;
+
+use SumoCoders\FrameworkMultiUserBundle\User\UserInterface;
+use SumoCoders\FrameworkMultiUserBundle\User\UserRepositoryCollection;
+
+class RequestPassword
+{
+    /**
+     * @var UserInterface
+     */
+    private $user;
+
+    /**
+     * @var UserRepositoryCollection
+     */
+    private $userRepositoryCollection;
+
+    /**
+     * RequestPassword constructor.
+     *
+     * @param UserRepositoryCollection $userRepositoryCollection
+     */
+    public function __construct(UserRepositoryCollection $userRepositoryCollection)
+    {
+        $this->userRepositoryCollection = $userRepositoryCollection;
+    }
+
+    /**
+     * @return UserInterface
+     */
+    public function getUser()
+    {
+        return $this->user;
+    }
+
+    /**
+     * @param $userName
+     */
+    public function setUser($userName)
```

Comment:
dit is voor een form kan enkel de setUser gebruiken, kan het enkel ergens anders gaan afhandelen